Source/WebCore/testing/Internals.idl

```diff
@@ -417,6 +417,16 @@ enum AV1ConfigurationRange {
     unsigned long attributes;
 };
 
+[
+    ExportMacro=WEBCORE_TESTSUPPORT_EXPORT,
+    JSGenerateToJSObject,
+] dictionary PDFAnnotationRect {
```

Comment:
Surprised we don't already have a recty type to use here

Reply:
`DOMRectReadOnly` comes to mind but it has a bit more information than you need :/